### DISCUSSION TITLE
z errors when I give the path to the directory

### DISCUSSION THREAD

**Discussion by Eshanatnight** [ajeetdsouza/zoxide#771] [Q&A] [answered]
I just installed zoxide, and am trying to go inside the dev directory. But I am getting this error.
I installed it with cargo(tried the brew version also it had the same issue)
I read the setup instructions, there was nothing like this. 
Can anyone help me with this?

```
z dev/
error: unrecognized subcommand 'dev/'

Usage: zoxide <COMMAND>

For more information, try '--help'.
```

EDIT: I closed and restarted the shell and it work

**Comment by ajeetdsouza** [comment]
I'm guessing you've aliased `z` to `zoxide`. The setup guide in the README has 4 steps, have you gone through all of them?

**Reply by Eshanatnight** [reply, marked as answer]
To be honest I have no Idea what happened. I installed it as per the readme. But then it was giving me that error. Then I closed the terminal and opened it again and it worked

**Comment by rjhoppe** [comment]
Encountering this same error and followed the steps in the readme. On WSL and did not install via the crossed-out Ubuntu link. Previously got the crash error as well.

**Reply by Eshanatnight** [reply]
I used homebrew for Linux to install zoxide, and it worked. So I can suggest that.

**Reply by ajeetdsouza** [reply]
@rjhoppe how exactly are you initialising zoxide?

**Reply by rjhoppe** [reply]
@ajeetdsouza I added eval "$(zoxide init bash)" to the bottom of my .bashrc file and I tried using the z and zi cmds. Aliasing to cmd consistently crashed my WSL session while the z and zi were not able to find any directory paths except when I input z without any directory path arguments and it functioned like an empty cmd command

**Reply by yxasika** [reply]
Same issue for me. I'm also running it in WSL Ubuntu in my Warp terminal. Couldn't do normal dir navigations.

I was installing it using the Ubuntu Linux commands. Reinstalling zoxide using linuxbrew made it work for me interestingly.
